packages/next/src/build/webpack/config/blocks/css/index.ts

```diff
@@ -619,7 +619,11 @@ export const css = curry(async function css(
           insert: function (linkTag: HTMLLinkElement) {
             if (typeof _N_E_STYLE_LOAD === 'function') {
               const { href, onload, onerror } = linkTag
-              _N_E_STYLE_LOAD(new URL(href).pathname).then(
+              _N_E_STYLE_LOAD(
+                href.indexOf(window.location.origin) === 0
+                  ? new URL(href).pathname
+                  : href
+              ).then(
```

Comment:
@sokra @ijjk I updated the fix to avoid a regression when the `assetPrefix` config is not used. I used the same kind of check than in the `MiniCssExtractPlugin` webpack plugin.